mapie/conformity_scores/sets/lac.py

```diff
@@ -0,0 +1,252 @@
+from typing import Optional, Union, cast
+
+import numpy as np
+
+from mapie.conformity_scores.classification import BaseClassificationScore
+from mapie.conformity_scores.sets.utils import check_proba_normalized
+from mapie.estimator.classifier import EnsembleClassifier
+
+from mapie._machine_precision import EPSILON
+from mapie._typing import ArrayLike, NDArray
+from mapie.utils import compute_quantiles
+
+
+class LAC(BaseClassificationScore):
+    """
+    Least Ambiguous set-valued Classifier (LAC) method-based
+    non conformity score (also formerly called ``"score"``).
+
+    It is based on the the scores (i.e. 1 minus the softmax score of the true
+    label) on the calibration set.
+
+    References
+    ----------
+    [1] Mauricio Sadinle, Jing Lei, and Larry Wasserman.
+    "Least Ambiguous Set-Valued Classifiers with Bounded Error Levels.",
+    Journal of the American Statistical Association, 114, 2019.
+
+    Attributes
+    ----------
+    classes: Optional[ArrayLike]
+        Names of the classes.
+
+    random_state: Optional[Union[int, RandomState]]
+        Pseudo random number generator state.
+
+    quantiles_: ArrayLike of shape (n_alpha)
+        The quantiles estimated from ``get_sets`` method.
+    """
+
+    def __init__(self) -> None:
+        super().__init__()
+
+    def set_external_attributes(
```

Comment:
is it necessary to re-define this method in the child class ?

Reply:
it's fine to leave it for future changes

Reply:
If in the future we need to redefine it, then we will rewrite it. However, I think to make the code more concise, we should remove it